npm start (Windows 7 0.0.0): Executed 55 of 59

PhantomJS 1.9.8 (Windows 7 0.0.0): Executed 55 of 59 (4 FAILED) (skipped 2) (0 sPhantomJS 1.9.8 (Windows 7 0.0.0): Executed 56 of 59 (4 FAILED) (skipped 2) (0 sPhantomJS 1.9.8 (Windows 7 0.0.0): Executed 57 of 59 (4 FAILED) (skipped 2) (0 sPhantomJS 1.9.8 (Windows 7 0.0.0): Executed 57 of 59 (4 FAILED) (skipped 2) (0.118 secs / 0.333 secs)

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "D:\\nodejs\\node.exe" "D:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "karma"
npm ERR! node v4.0.0
npm ERR! npm  v2.14.2
npm ERR! code ELIFECYCLE
npm ERR! react-motion@0.3.0 karma: `set NODE_ENV=development && karma start ./karma.conf.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-motion@0.3.0 karma script 'set NODE_ENV=development && karma start ./karma.conf.js'.
npm ERR! This is most likely a problem with the react-motion package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     set NODE_ENV=development && karma start ./karma.conf.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls react-motion
npm ERR! There is likely additional logging output above.

--- a/karma.conf.js
+++ b/karma.conf.js
@@ -1,5 +1,7 @@
 'use strict';
 
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
 var path = require('path');
 var webpack = require('webpack');
 

--- a/karma.conf.js
+++ b/karma.conf.js
@@ -5,13 +5,15 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'development';
 var path = require('path');
 var webpack = require('webpack');
 
+var withCoverage = process.argv.indexOf('coverage') !== -1 || process.env.COVERAGE;
+
 var webpackConfig = {
   devtool: 'eval',
   resolve: {
     extensions: ['', '.js'],
   },
   module: {
-    loaders: process.env.COVERAGE ?
+    loaders: withCoverage ?
       [
         {test: /\.js$/, loader: 'babel', include: [path.resolve('./test')]},
         {test: /\.js$/, loader: 'isparta', include: [path.resolve('./src')]},

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "test": "npm run karma -- --single-run --reporters nyan",
     "test:travis": "npm run karma -- --single-run",
     "test:dev": "npm run karma -- --no-single-run --auto-watch --reporters nyan",
-    "test:cov": "COVERAGE=1 npm run karma -- --single-run --reporters coverage"
+    "test:cov": "npm run karma -- --single-run --reporters coverage"
   },
   "repository": {
     "type": "git",

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "start": "node server.js",
     "build-demos": "webpack",
     "lint": "eslint --ext .js,.jsx .",
-    "prerelease": "babel src --out-dir lib && babel native/non-compiled.js > native/index.js && NODE_ENV=production webpack --config webpack.prod.config.js",
+    "prerelease": "babel src --out-dir lib && babel native/non-compiled.js > native/index.js && webpack --config webpack.prod.config.js",
     "karma": "NODE_ENV=development karma start ./karma.conf.js",
     "test": "npm run karma --  --single-run --reporters nyan",
     "test:travis": "npm run karma --  --single-run",

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "webpack-dev-server": "^1.10.1"
   },
   "scripts": {
-    "start": "NODE_ENV=development node server.js",
+    "start": "node server.js",
     "build-demos": "NODE_ENV=production webpack",
     "lint": "eslint --ext .js,.jsx .",
     "prerelease": "babel src --out-dir lib && babel native/non-compiled.js > native/index.js && NODE_ENV=production webpack --config webpack.prod.config.js",

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
   },
   "scripts": {
     "start": "node server.js",
-    "build-demos": "NODE_ENV=production webpack",
+    "build-demos": "webpack",
     "lint": "eslint --ext .js,.jsx .",
     "prerelease": "babel src --out-dir lib && babel native/non-compiled.js > native/index.js && NODE_ENV=production webpack --config webpack.prod.config.js",
     "karma": "NODE_ENV=development karma start ./karma.conf.js",

--- a/package.json
+++ b/package.json
@@ -38,9 +38,9 @@
     "build-demos": "webpack",
     "lint": "eslint --ext .js,.jsx .",
     "prerelease": "babel src --out-dir lib && babel native/non-compiled.js > native/index.js && webpack --config webpack.prod.config.js",
-    "karma": "NODE_ENV=development karma start ./karma.conf.js",
-    "test": "npm run karma --  --single-run --reporters nyan",
-    "test:travis": "npm run karma --  --single-run",
+    "karma": "karma start ./karma.conf.js",
+    "test": "npm run karma -- --single-run --reporters nyan",
+    "test:travis": "npm run karma -- --single-run",
     "test:dev": "npm run karma -- --no-single-run --auto-watch --reporters nyan",
     "test:cov": "COVERAGE=1 npm run karma -- --single-run --reporters coverage"
   },

--- a/server.js
+++ b/server.js
@@ -1,3 +1,5 @@
+'use strict';
+
 var webpack = require('webpack');
 var WebpackDevServer = require('webpack-dev-server');
 var config = require('./webpack.config');

--- a/server.js
+++ b/server.js
@@ -3,6 +3,8 @@ var WebpackDevServer = require('webpack-dev-server');
 var config = require('./webpack.config');
 var port = process.env.PORT || 3000;
 
+process.env.NODE_ENV = 'development';
+
 new WebpackDevServer(webpack(config), {
   publicPath: config.output.publicPath,
   hot: true,

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,3 +1,5 @@
+'use strict';
+
 process.env.NODE_ENV = process.env.NODE_ENV || 'production';
 
 var webpack = require('webpack');

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,3 +1,5 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'production';
+
 var webpack = require('webpack');
 
 var loaders = ['babel'];

--- a/webpack.prod.config.js
+++ b/webpack.prod.config.js
@@ -1,3 +1,5 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'production';
+
 var webpack = require('webpack');
 var path = require('path');
 

--- a/webpack.prod.config.js
+++ b/webpack.prod.config.js
@@ -1,3 +1,5 @@
+'use strict';
+
 process.env.NODE_ENV = process.env.NODE_ENV || 'production';
 
 var webpack = require('webpack');

